packages/build/webpack/constants.js

```diff
@@ -177,14 +191,16 @@ const labkeyPackagesDev = process.env.LINK
         '@labkey/premium/entities': labkeyUIPremiumPath + '/entities',
         '@labkey/premium/workflow': labkeyUIPremiumPath + '/workflow',
         '@labkey/premium/storage': labkeyUIPremiumPath + '/storage',
-        '@labkey/premium/search': labkeyUIPremiumPath + '/search',
```

Comment:
did you mean to remove the `@labkey/premium/search` line here?

Reply:
thanks Cory, this is fixed